LazyInterface/LazyInterface/ILazy.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="ILazy.cs" company="Kalinin Andrew">
+// Copyright (c) Kalinin Andrew. All rights reserved.
+// </copyright>
+
+namespace LazyInterface
+{
```

Comment:
используйте file-scoped неймспейсы
```suggestion
namespace LazyInterface;
```